### ISSUE TITLE
Disappearing TinyMCE buttons in Umbraco 13 (upgraded from 12.3.3, tried also in 13.1.0 RC)

### ISSUE THREAD

**Issue by Baracuda011** [umbraco/Umbraco-CMS#15581]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

13.1.0 RC

### Bug summary

https://github.com/umbraco/Umbraco-CMS/assets/14230396/0c17ecda-b0b1-42e3-bd7d-641d4e788bbc

I have a lot of migrated static RTE areas that are missing all the RTE controls. There is nothing too specific related to this content besides a few CSS classes. In the attached video you will see the behaviour I am getting:

1. When I click into the RTE field all the controls are missing
2. When I CUT the content and PASTE back, the buttons appear
3. After I save the field and open it again, the controls are missing again

When I create RTE on the new and empty page, everything works fine. But if I add a new RTE on the page that already contains some, this new RTE behaves in the same way.

### Specifics

https://github.com/umbraco/Umbraco-CMS/assets/14230396/0c17ecda-b0b1-42e3-bd7d-641d4e788bbc

### Steps to reproduce

https://github.com/umbraco/Umbraco-CMS/assets/14230396/0c17ecda-b0b1-42e3-bd7d-641d4e788bbc

The website is upgraded from 12.3.3 to 13.0.3 initially. When I saw this RTE issue I have tried with 13.1.0 RC as I saw some comments about RTE fixes coming in 13.1.0. This issue still exists in 13.1.0. RC.

1. I click into the RTE field, all the controls are missing
2. CUT the content, PASTE it back, the buttons appear
3. Save the field and open it again, the controls are missing again

### Expected result / actual result

The expected result is the normal behaviour and appearance of the RTE control. Instead, I am getting an empty textarea.

**Comment by iOvergaard** [comment]
Hi @Baracuda011 

I believe this might be a duplicate of #15515 and as such, there is now a pending pull request to fix the problem. Do you believe that #15515 adequately describes the problems you are seeing here or is there more to it?

**Comment by iOvergaard** [comment]
This should be fixed with #15595 which we expect to release with Umbraco 13.2. Thanks for reporting this! #h5yr

**Comment by Baracuda011** [comment]
Thanks, @iOvergaard for the confirmation.

I hoped this could pop in a sooner release, but better late than never. Is there a date for 13.2?

**Comment by iOvergaard** [comment]
@Baracuda011 You are right, it should be 13.1 instead. I changed the target; in that case, it should be released next week.

**Comment by Baracuda011** [comment]
@iOvergaard WOO-HOO!!!
